gtfs/download_gtfs_feeds.py

```diff
@@ -47,7 +52,7 @@ def gtfs_line_intersects(name_row, data_row, bbox):
   csvlines = [line.rstrip().split(':') for line in csvlines]
 
 for city_bbox in csvlines:
-  if city_bbox[0] == sys.argv[1]:
+  if city_bbox[0] == os.environ["CITY_NAME"]:
```

Comment:
Is there any specific reason why this is a distinct variable instead of reusing existing `HEADWAY_AREA`?